### DISCUSSION TITLE
IMHO pasting a clipped element onto the same layer is weird

### DISCUSSION THREAD

**Discussion by fmaida** [PintaProject/Pinta#641] [Ideas]
I don't know how graphics applications work these days on Windows and Linux, but generally on Mac when you crop a portion and then paste it, the crop is pasted into a new layer (at least this is the standard behavior of Pixelmator, Pixelmator Pro and Serif Affinity Photo)

![ScreenFlow](https://github.com/PintaProject/Pinta/assets/605017/45df082e-85fe-4eac-bdb6-36f5f8b1e13d)

For this reason when I cut and paste I found it a little strange to find the cutting added again on the same layer I was working in.

I realize that perhaps on other operating systems you are accustomed differently, so I would suggest inserting an option in which you can specify what behavior the program must have when a paste is made (same layer vs new layer).

Thank you

**Comment by cameronwhite** [comment]
There is `Edit / Paste Into New Layer` (Cmd + Shift +V) which does what you're after

**Comment by fmaida** [comment]
> There is `Edit / Paste Into New Layer` (Cmd + Shift +V) which does what you're after

Yes, I saw it. But I usually use keyboard shortcuts to cut, copy and paste. If it is possible and easy to implement it in the future, it would be nice if it were possible to choose in the options the default paste mode, which is the one that is invoked when you press CTRL+V or CMD+V.

**Comment by mlntr** [comment]
This behaviour surprised me also. Was a slight annoyance to resort to manually creating a new layer each time, then paste (an image from anywhere).
